templates/README.tpl.md

```diff
@@ -152,6 +145,7 @@ Cozy <APP_NAME> is developed by Cozy Cloud and distributed under the [AGPL v3 li
 [cozy]: https://cozy.io "Cozy Cloud"
 [setup]: https://dev.cozy.io/#set-up-the-development-environment "Cozy dev docs: Set up the Development Environment"
 [agpl-3.0]: https://www.gnu.org/licenses/agpl-3.0.html
+[contribute]: https://github.com/cozy/<SLUG_GH>/blob/master/CONTRIBUTING.md
```

Comment:
A relative link to CONTRIBUTING.md is enough, I think. And it will avoid the risk to forget to replace the slug.